data/test_endpoints/event_brokers/kafka_sasl_plaintext_endpoint.yml

```diff
@@ -3,5 +3,6 @@ event_broker:
   security_protocol: SASL_PLAINTEXT
   topic: topic
   url: localhost
+  partition_key: foo
```

Comment:
can we please also document this param in the documentation?

Reply:
Yes, I've updated the Kafka docs to describe the partition param.